test/functional/pdo_sqlsrv/pdo_empty_result_error.phpt

```diff
@@ -0,0 +1,98 @@
+--TEST--
+Test that calling nextRowset() on an empty result set produces the correct error message. Fix for Github 507.
+--SKIPIF--
+<?php require('skipif.inc'); ?>

```

Comment:
Please try to limit the length of test title. If you have more please put it in --DESCRIPTION-- instead.
For tests, it's a good practice to drop test tables / procedures at the end to leave no trace.

Reply:
I do drop them at the end, but I kept the drop functions at the beginning too just in case...

Reply:
ahhh... I see! In the beginning you use DropTable and DropProc but later you simply use queries.